ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -416,7 +433,7 @@ def process(self, device, results, unused):
                     set_image=prepId('image-{0}'.format(image_id)),       # image-346eeba5-a122-42f1-94e7-06cb3c53f690
                     set_tenant=prepId('tenant-{0}'.format(tenant_id)),    # tenant-a3a2901f2fd14f808401863e3628a858
                     hostId=server.get('hostId', ''),                      # a84303c0021aa53c7e749cbbbfac265f
-                    hostName=server.get('name', '')                       # cloudserver01
+                    hostName=server.get('OS-EXT-SRV-ATTR:hypervisor_hostname', '')                       # cloudserver01
```

Comment:
Whoah.  You're changing the meaning of this attribute.  It is supposed to be the hostname of the instance, not of the hypervisor host!